syntax = "proto3";

package echo;

service EchoService {
  rpc sendStats (statsNameParam) returns (resultValue) {}
  rpc sendMetrics (statsNameParam) returns (resultValue) {}
  rpc getNumberOfViolationsPast (listMetricsParam) returns (resultValue) {}
  rpc getNumberOfViolationsFuture (listMetricsParam) returns (resultValue) {}
}

message resultValue {
  string result = 1;
}

message statsNameParam {
  string statsName = 1;
}

message listMetricsParam {
  string listMetrics = 1;
}

message emptyParam {}